tgui/packages/tgui-panel/settings/constants.js

```diff
@@ -14,3 +14,20 @@ export const SETTINGS_TABS = [
     name: 'Chat Tabs',
   },
 ];
+
+
+export const FONTS_DISABLED = "Use CSS";
```

Comment:
```suggestion
export const FONTS_DISABLED = "Default";
```

I don't like this exposing of implementation details to players.